csrankings-v.csv

```diff
@@ -215,6 +215,7 @@ Vijay Kumar 0001,University of Pennsylvania,http://www.kumarrobotics.org,h6h5Fdo
 Vijay Kumar 0002,University of Missouri - Kansas City,http://k.web.umkc.edu/kumarv/index.html,NOSCHOLARPAGE
 Vijay Nagarajan,University of Edinburgh,http://homepages.inf.ed.ac.uk/vnagaraj,xQYIFX0AAAAJ
 Vijay Natarajan,IISc Bangalore,http://www.csa.iisc.ac.in/~vijayn,yePyztMAAAAJ
+Vijay V Raghavan,University of Louisiana - Lafayette,https://people.cmix.louisiana.edu/vijay/,GFW9JacAAAAJ

```

Comment:
Needs two entries, one with each correct suffix: Vijay Raghavan 0001, Vijay V. Raghavan 0001

Reply:
Added Both entries with suffix 0001 for Dr Vijay Raghavan